test/extended/olm/olmv1.go

```diff
@@ -121,6 +121,7 @@ var _ = g.Describe("[sig-olmv1] OLMv1 operator installation", func() {
 	})
 
 	g.It("should install a cluster extension", func(ctx g.SpecContext) {
+		g.Skip("Test is temporarily disabled")
```

Comment:
Can we say something like 
```suggestion
		g.Skip("Test is temporarily disabled while we make anticipated, breaking changes.")
```
?